recipe/meta.yaml

```diff
@@ -12,22 +12,24 @@ source:
     - 579-set-modification-time-for-ccahe-file-for-mamba.patch
 
 build:
-  number: 1
+  noarch: python
+  number: 2
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
     - constructor = constructor.main:main
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
+    - __{{ target_os }}
     - conda >=4.6
-    - python
+    - python >=3.6
     - ruamel_yaml >=0.11.14,<0.16
     - conda-standalone
     - pillow >=3.1  # [win]
-    - nsis >=3.01  # [win]
+    - nsis >=3.01,<3.08  # [win]
```

Comment:
Why is the upper bound here needed?

Reply:
Mitigation for https://github.com/conda/constructor/issues/526

We patched it in repodatas too: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/300